xbmc/cores/VideoPlayer/DVDCodecs/Video/VDPAU.cpp

```diff
@@ -2450,6 +2486,9 @@ void CMixer::InitCycle()
 
   m_processPicture.DVDPic = m_mixerInput[1].DVDPic;
   m_processPicture.videoSurface = m_mixerInput[1].videoSurface;
+
+  // update deinterlacing method used in processInfo (none if progressive)
+  m_config.processInfo->SetVideoDeintMethod(deintStr);
```

Comment:
Ups. Leftover. Something for tonight.

Am 29.08.2016 5:46 nachm. schrieb "Rainer Hochecker" <
notifications@github.com>:

> In xbmc/cores/VideoPlayer/DVDCodecs/Video/VDPAU.cpp
> <https://github.com/FernetMenta/xbmc/pull/404#discussion_r76631096>:
>
> > @@ -2450,6 +2486,9 @@ void CMixer::InitCycle()
> >
> >    m_processPicture.DVDPic = m_mixerInput[1].DVDPic;
> >    m_processPicture.videoSurface = m_mixerInput[1].videoSurface;
> > +
> > +  // update deinterlacing method used in processInfo (none if progressive)
> > +  m_config.processInfo->SetVideoDeintMethod(deintStr);
>
> you still call m_config.processInfo->SetVideoDeintMethod in InitCycle,
> which is wrong
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/FernetMenta/xbmc/pull/404/files/9b3f4db0ce2a8efa2228def0dd191ba5c5e80e9e#r76631096>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABCfHVN-318HrsmHP-VPrmA6oRo3Xv3gks5qkv7jgaJpZM4JteUI>
> .
>